docs/resources/examples/repeater_client.yaml

```diff
@@ -0,0 +1,68 @@
+#######################################
+# ROS DISCOVERY SERVER CLIENT EXAMPLE #
+#######################################
+
+##################################
+# CONFIGURATION VERSION
+version: v3.0                                                     # 0
+
+##################################
+# ALLOWED TOPICS
+# Allowing ROS2 HelloWorld demo_nodes
+
+allowlist:
+  - name: rt/chatter                                              # 1
+    type: std_msgs::msg::dds_::String_                            # 1
+
+##################################
+# PARTICIPANTS
+participants:
+
+##################################
+# SIMPLE PARTICIPANT
+# This participant will subscribe to topics in allowlist in domain 0 and listen every message published there
+
+  - name: SimpleROS2                                              # 2
+    kind: local                                                   # 3
+    domain: 0                                                     # 4
+
+##################################
+# ROS DISCOVERY CLIENT
+# This participant will subscribe to topics in allowlist using Discovery Server protocol as Super Client
+
+  - name: Client                                                  # 5
+    kind: wan                                                     # 6
+    connection-addresses:
+        addresses:
+          - domain: 1.1.1.1
+            port: 11666
+            transport: tcp
+
+
+##################################
+# CONFIGURATION DESCRIPTION
+
+# This configuration example configures a DDS Router to listen to every message published in domain 0 in topic
+# rt/chatter from ROS2 demo_nodes, and to transmit these messages through a Discovery Server (configured as
+# Super Client) to another Discovery Server (configured as Server).
```

Comment:
Update. Some other yaml descriptions require review (like mentioning super client in initial peers configs).